cocotb/share/makefiles/simulators/Makefile.verilator

```diff
@@ -0,0 +1,96 @@
+###############################################################################
+# Copyright (c) 2013 Potential Ventures Ltd
+# Copyright (c) 2019 Stefan Wallentowitz
```

Comment:
I'd prefer if we can stick to the standard header unmodified, even though it's not fully correct. I hope to get rid of those verbose headers in #804, and all special cases make this task even more annoying than it already is.